src/Invictus.Testing/LogicAppConverter.cs

```diff
@@ -0,0 +1,99 @@
+﻿using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Linq;
+using Invictus.Testing.Model;
+using Microsoft.Azure.Management.Logic.Models;
+using Newtonsoft.Json;
+
+namespace Invictus.Testing
+{
+    internal class LogicAppConverter
```

Comment:
I think we really need unit tests for these :)

Reply:
Done.